qutip/mcsolve.py

```diff
@@ -80,10 +74,9 @@ def step(self, *args):
         self.call_args[2] = itask
         return r
 
-
-def mcsolve(H, psi0, tlist, c_ops=[], e_ops=[], ntraj=None,
-            args={}, options=None, progress_bar=True,
-            map_func=None, map_kwargs=None,
+def mcsolve(H, psi0, tlist, c_ops=[], e_ops=[], ntraj=0,
+            args={}, options=Options(),
+            progress_bar=True, map_func=parallel_map, map_kwargs={},
```

Comment:
`map_func` default changed from `None` to `parallel_map `. @fminga is this related to that modification you were proposing of avoiding parallel for one trajectory?

Reply:
More or less, my point is that the default `map_func` is always `parallel_map`. Probably we can change it to `serial_map` if `ntraj=1`. This would allow to easily run in parallel several different single trajectories (for example, with different parameters) with `parfor` or `parallel_map`.

Reply:
Do it later at line 413.